mmv1/third_party/terraform/services/netapp/resource_netapp_kmsconfig_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+// ----------------------------------------------------------------------------
+//
+//     ***     AUTO GENERATED CODE    ***    Type: MMv1     ***
+//
+// ----------------------------------------------------------------------------
+//
+//     This file is automatically generated by Magic Modules and manual
+//     changes will be clobbered when the file is regenerated.
+//
+//     Please read more about how to change this file in
+//     .github/CONTRIBUTING.md.
+//
+// ----------------------------------------------------------------------------
+
+package netapp_test
+
+import (
+	// "fmt"
+	// "strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	// "github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	// "github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	// transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func TestAccNetappkmsconfig_kmsConfigCreateExample_Update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetappkmsconfig_kmsConfigCreateExample(context),
```

Comment:
this is referencing the generated test -- did you mean to reference the `testAccNetappkmsconfig_kmsConfigCreateExample_Full` below (which is currently unused)?